Dockerfile

```diff
@@ -32,12 +32,10 @@ ARG DEBIAN_FRONTEND=noninteractive
 # We need the entire Runstone server repo for the build.
 COPY . ${RUNESTONE_PATH}
 RUN \
```

Comment:
OK, so if we have DOCKER_BUILD_ARGS set -- we can still use `docker build....` right?

Reply:
It's possible to invoke `docker-build` directly, but does require an extra param (the `DOCKER_BUILD_ARGS`).

Reply:
...but the build process requires the entire repo to be present, so this is probably a non-starter...